config/config/wc-config.yaml

```diff
@@ -256,7 +256,6 @@ fluentd:
   user:
     resources:
       limits:
-        cpu: 500m
```

Comment:
I do not think we want this. as fluentd can use the entire cpu and affected other pods and services